api.bs

```diff
@@ -2338,6 +2338,45 @@ for a number of reasons:
     Without allocating [=privacy budget=] for new data,
     sites could exhaust their budget forever.
 
+### Formal Analysis of Privacy Properties and Their Limitations ### {#formal-analysis}
```

Comment:
I think that you want to mention both papers in the introduction to this, not just the big bird one.  Point out that the first establishes the basic system that this document relies on and the second expands that to be a more comprehensive analysis of the system as a whole and the overall privacy guarantees.

This is a timely reminder that we need to implement safety limits in the spec.